modules/docs/lib/widgets/enhancedComponent.tsx

```diff
@@ -0,0 +1,199 @@
+ * ```
+ */
+const ParentComponentJSDocContext = React.createContext(defaultJSDoc);
+
+const fileNameToCategoryMap: Record<string, string[]> = {
+  Layout: [
+    'utils/layout',
+    'utils/gridArea',
+    'utils/gridItem',
+    'utils/flexItem',
+    'utils/position',
+    'utils/space',
+    'utils/stack',
+  ],
+  Color: ['utils/color', 'utils/background'],
+  Border: ['utils/border'],
+  Depth: ['utils/depth'],
+  Text: ['utils/text'],
+  Other: ['utils/other'],
+};
+
+registerWidget<EnhancedComponentValue>('enhancedComponent', ({value, doc, meta}) => {
+  const groups = groupProps(value.props);
+  const parentComponentName = React.useContext(ParentComponentNameContext);
+  const parentComponentJSDoc = React.useContext(ParentComponentJSDocContext);
+  const hideHeader = meta && meta.hideHeader === true;
+
+  const intro = (
+    <>
+      {!parentComponentName && !hideHeader && value.displayName ? (
+        <Heading>{value.displayName}</Heading>
+      ) : null}
+
+      <ParentComponentNameContext.Provider value="">
+        <ParentComponentJSDocContext.Provider value={defaultJSDoc}>
+          <MdxJSToJSX>{parentComponentJSDoc.description || doc?.description || ''}</MdxJSToJSX>
+        </ParentComponentJSDocContext.Provider>
+      </ParentComponentNameContext.Provider>
+    </>
+  );
+
+  // We don't want to re-document a subcomponent that is a container component with a different
+  // model. If we detect another container, only render the intro and a link to the container
+  // component
+  if (parentComponentName && value.componentType === 'container') {
+    return (
+      <>
+        {intro}
+        <MDX as="p">
+          This component references the{' '}
+          <ParentComponentNameContext.Provider value="">
+            <ParentComponentJSDocContext.Provider value={defaultJSDoc}>
+              <SymbolDialog value={{kind: 'symbol', name: value.displayName || ''}} />
+            </ParentComponentJSDocContext.Provider>
+          </ParentComponentNameContext.Provider>{' '}
+          component.
+        </MDX>
+      </>
+    );
+  }
+
+  return (
+    <>
+      {intro}
+      {value.styleComponent ? (
+        <>
+          <Heading headingOffset={1}>Style Component</Heading>
+          <MDX as="p">
+            <code>
+              <ParentComponentJSDocContext.Provider value={defaultJSDoc}>
+                <Value value={value.styleComponent} />
+              </ParentComponentJSDocContext.Provider>
+            </code>
+          </MDX>
+        </>
+      ) : null}
+      <Heading headingOffset={1}>Props</Heading>
+      {value.baseElement && (
+        <MDX as="p">
+          Props extend from <Value value={value.baseElement} />. Changing the <code>as</code> prop
+          will change the element interface.
+        </MDX>
+      )}
+      {value.componentType === 'container' && value.model ? (
+        <MDX as="p">
+          Props extend from <SymbolDialog value={{kind: 'symbol', name: `${value.model}Config`}} />.
```

Comment:
Trying to figure out why the contents of this `SymbolDialog` for container components are empty on the Canvas Site:

<img width="442" alt="image" src="https://user-images.githubusercontent.com/146020/220756296-ab493db5-4610-494b-b19e-c5a32d6f963e.png">

It renders fine in Storybook:

<img width="1436" alt="image" src="https://user-images.githubusercontent.com/146020/220756442-ff7f723e-9a82-45cb-8f9f-4057d1716a70.png">

Reply:
Looks like the `ActionBarModelConfig` `properties` is an empty array in the built `dist/es6/lib/docs.js` file used by the site...
```
{
  "name": "ActionBarModelConfig",
  "packageName": "react",
  "fileName": "/Users/james.fan/Projects/dse/canvas-kit/modules/react/action-bar/lib/useActionBarModel.tsx",
  "description": "",
  "declarations": [
    {
      "name": "useActionBarModel",
      "filePath": "/Users/james.fan/Projects/dse/canvas-kit/modules/react/action-bar/lib/useActionBarModel.tsx"
    }
  ],
  "tags": {},
  "type": {
    "kind": "object",
    "properties": []
  }
},
```
... whereas the `ActionBarModelConfig` item included in `window.__docs` referenced by Storybook has 64 `properties`:

<img width="367" alt="image" src="https://user-images.githubusercontent.com/146020/220793007-a5e46e9a-e803-4cdc-aebc-b5d537569b82.png">

Reply:
Fixed